test/e2e/cypress/support/commands.js

```diff
@@ -155,3 +155,13 @@ Cypress.Commands.add("RunSQL", (folder, files, fail = true) => {
       });
    });
 });
+
+Cypress.Commands.add("TestLog", (log) => {
+   // Have the running services clear their definitions.
+   cy.request("POST", "/testlog", { log });
+});
+
+Cypress.Commands.add("VersionCheck", () => {
+   // Have the running services clear their definitions.
```

Comment:
maybe, update the comment

Reply:
OK, yeah that is nice.  I was looking for something like `Cypress.currentTest.title` but didn't find it.  

I'm making that change